TravelBug/src/Components/VerticalCard/VerticalCard.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import "./VerticalCard.css";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { useAuth } from "../../Contexts/Authcontext";
+import { useData } from "../../Contexts/Datacontext";
+const VerticalCard = ({ vid }) => {
+  const { token } = useAuth();
+  const { state, dispatch } = useData();
+  const navigate = useNavigate();
+  const [appear, setAppear] = useState(false);
+  const { title, category, img, creator, _id } = vid;
+  const toSingleVideoPage = () => {
+    navigate(`/singlevideo/${_id}`);
+  };
+
+  const deleteLikeHandler = async () => {
+    try {
+      const res = await axios.delete(`/api/user/likes/${_id}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log("del response ", res);
```

Comment:
- you can remove this console, i think it was for testing purpose only

Reply:
Done